addons/mail/static/src/messaging/entity/chatter/chatter.js

```diff
@@ -47,13 +55,19 @@ function ChatterFactory({ Entity }) {
             return this.activities.filter(activity => activity.state === 'overdue');
         }
 
-        refresh() {
+        async refresh() {
             const thread = this.thread;
             if (!thread || thread.isTemporary) {
                 return;
             }
             thread.loadNewMessages();
-            thread.fetchAttachments();
+            if (this.env.disableLoadingDelay) {
+                this.isShowingAttachmentsLoading = true;
```

Comment:
I'm not sure but wouldn't it be better if the code was the same as before, but the only effect of `this.env.disableLoadingDelay` being to set the timeout to 0 instead of 400?

And then I would call it sometime like `showLoadingDelay` and also use it in the other places of the code doing the same.

Reply:
(so give it 400 by default and 0 in tests)

Reply:
@alexkuhn did not agree to use a variable and preferred using 400 in a "hardcoded" way so I don't which one I should listen ?

And for the timeout 0 I thought it was a little bit useless to create timeout just to create one, but I think this could result to be slightly the same

Reply:
`setTimeout(0)` in tests still make it asynchronous, and it's bad if we have to `await` a tick just because of implementation detail that makes "instantaneous" not really instantaneous...

Reply:
Except in practice it is never instantaneous, since we do loading stuff like this in the context of async stuff/RPC anyway, and `afterNextRender` is already taking care of ignoring such extra delay.

This is actually a step in the right direction to make the test closer to what actually happens in practice without wasting time waiting on huge delay. I think this is the best of both.

Reply:
So, what is the consensus :3 ?

Reply:
Does it work without any other change with timeout 0, or are other changes needed for it to work?

Reply:
I'll check that

Reply:
Slight changes in tests but no big deals